changelog/v1.5.0-beta21/failover-dns-docs.yaml

```
@@ -0,0 +1,7 @@
+changelog:
+- type: FIX
```

not really a user-facing change that needs to be called out in the changelog, imo
```suggestion
- type: NON_USER_FACING
```